internal/host/installcmd.go

```diff
@@ -80,6 +80,21 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 		data["INSTALLATION_TIMEOUT"] = strconv.Itoa(int(i.instructionConfig.InstallationTimeout))
 	}
 
+	if cluster.HTTPProxy != "" || cluster.HTTPSProxy != "" {
+		if cluster.HTTPProxy != "" {
+			cmdArgsTmpl = cmdArgsTmpl + " --http-proxy {{.HTTP_PROXY}}"
+			data["HTTP_PROXY"] = cluster.HTTPProxy
+		}
+		if cluster.HTTPSProxy != "" {
+			cmdArgsTmpl = cmdArgsTmpl + " --https-proxy {{.HTTPS_PROXY}}"
+			data["HTTPS_PROXY"] = cluster.HTTPSProxy
+		}
+		if cluster.NoProxy != "" {
```

Comment:
You can have proxy and no proxy at the same time?

Reply:
yes, no_proxy excludes the specified list of domains/CIDR from being proxified when http_proxy/https_proxy is configured